pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1165,7 +1165,10 @@ self: super: {
   cabal-install-parsers = dontCheck super.cabal-install-parsers;
 
   # gitit is unbroken in the latest release
-  gitit = markUnbroken super.gitit;
+  # version constraints break the build, so
+  # we jailbreak until the next release
+  # to avoid to add >= 4 patches
+  gitit = doJailbreak (markUnbroken super.gitit);
```

Comment:
Instead of using `markUnbroken` here, can you remove `gitit` from the list of broken packages in `configuration-hackage2nix.yaml`?  The `hackage-packages.nix` is regenerated everyday automatically, so as long as `gitit` is not in the list of broken packages, it will be regenerated without `broken = true;`.

Also, if you could link to and upstream issue about the incorrect version constraints, that would be helpful (so that the Haskell maintainers here in nixpkgs will know when the `doJailbreak` can be removed).  If this is already fixed upstream (and a release just needs to be made), then just leave the expected version number in a comment above this line about when you expect the `doJailbreak` can be removed.

Reply:
Done!